### ISSUE TITLE
not enough swap space issue

### ISSUE THREAD

**Issue by ggbetz** [logikon-ai/cot-eval#10]
when evaluating microsoft/orca-7b

**Comment by ggbetz** [comment]
see also https://github.com/vllm-project/vllm/issues/787

There's env variable to set that, so we have to add, e.g.

```
VLLM_SWAP_SPACE=6
```

To the configuration (e.g., `config.env`)

**Comment by ggbetz** [comment]
The problem seems to arise when traces are generated with beam search.
For clarification: https://github.com/vllm-project/vllm/issues/2853

**Comment by yakazimir** [comment]
Might not be related, but I'm not getting this error when running our pipeline from before: 
```
2024-03-18T23:21:47.261237923Z All rights reserved.
2024-03-18T23:21:47.263064047Z 
2024-03-18T23:21:47.263076221Z Various files include modifications (c) NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
2024-03-18T23:21:47.263080129Z 
2024-03-18T23:21:47.263082384Z This container image and its contents are governed by the NVIDIA Deep Learning Container License.
2024-03-18T23:21:47.263085299Z By pulling and using the container, you accept the terms and conditions of this license:
2024-03-18T23:21:47.263087714Z https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license
2024-03-18T23:21:47.465183995Z 
2024-03-18T23:21:47.465206420Z NOTE: CUDA Forward Compatibility mode ENABLED.
2024-03-18T23:21:47.465212823Z   Using CUDA 12.2 driver version 535.104.05 with kernel driver version 525.147.05.
2024-03-18T23:21:47.465217873Z   See https://docs.nvidia.com/deploy/cuda-compatibility/ for details.
2024-03-18T23:21:47.508137755Z 
2024-03-18T23:21:47.527341591Z cp: target './cot-eval-cache/eleuther/tasks/logikon' is not a directory
```

**Comment by ggbetz** [comment]
This has been resolved.

Pipeline ran successfully for  the `NousResearch/Nous-Hermes-llama-2-7b` model with latest docker container.

Increasing the VLLM_SWAP_SPACE, i.e., the CPU memory vllm may use for offloading during beam search, resolves this issue.

Details:
* On slurm, I had to make sure to grab not only GPUs, but also enough memory allocated to the job via `ntasks`parameter. Before, I got OOM (not cuda oom) error.
* I used `VLLM_SWAP_SPACE=32`
